src/App.tsx

```diff
@@ -8,7 +12,71 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+type ActiveModal = {
+  selectedTodo: Todo | null,
+  isActive: boolean,
+};
+
+const filterTodos = (todos: Todo[], filterBy: string, search: string) => {
+  return todos.filter(({ completed }) => {
```

Comment:
you can just return initial list of todos if filter type is "all".  it will be more efficient